OniProj/OniCMakeProjs/OniProj/CMakeLists.txt

```diff
@@ -547,9 +547,22 @@ elseif(APPLE)
             ../../OniGameSource/Platform_MacOS/Oni_Platform_Mac.h)
 endif()
 
+set(ONI_MAP_PATH $<TARGET_FILE_DIR:Oni>/$<TARGET_FILE_NAME:Oni>.map)
+
 target_precompile_headers(Oni PRIVATE ../../../BungieFrameWork/BFW_Headers/BFW_MasterHeader.h)
-target_compile_definitions(Oni PRIVATE ONI_MAP_FILE="OniDbg.map")
+target_compile_definitions(Oni
+	PRIVATE
```

Comment:
The CMakeLists.txt committed by @hogsy used 8-space indentation. Tabs are added here.
With #1 I've accidentally added some 4-space indentations.

Reply:
In https://github.com/hogsy/OniFoxed/pull/3, I've added a `.editorconfig` to get some IDE support.
I configured it to use hard tabs everywhere, unless a exceptions are added (as I did here for all `.yml` files).

In @hogsy 's original cmake projects, the root cmake script has hard tabs, but the sub-cmake scripts have soft tabs.
In #3, I've opted to always use hard tabs (both for c/h sources and for cmake scripts).

So currently, the cmake scripts have an inconsistent formatting (hard tabs + soft tabs).
That's why I suggest in #7 to reformat all sources in one pr. Go for the short pain.

Reply:
I'm asking for this because my IDE did auto-reformat BFW.h in this pr.

Reply:
When this pr gets accepted, I will open a new pr reformatting/refactoring cmake + adding some options related to tool/debug/...

Also, I think we should aim to move all other build scripts (=vcxprojs + dsp + ...) to cmake.